config.json

```diff
@@ -0,0 +1,153 @@
+      "values": ["AnyOS", "Windows_NT", "Linux", "OSX", "FreeBSD", "NetBSD"],
+      "defaultValue": "AnyOS"
+     },
+    "RestoreDuringBuild": {
+      "description": "Enables/Disables the restoration of packages.",
+      "valueType": "property",
+      "values": ["True", "False"],
+      "defaultValue": true
+    },
+    "SkipTests":{
+      "description": "Enables/Disables running tests.",
+      "valueType": "property",
+      "values": ["True", "False"],
+      "defaultValue": false
+    },
+    "MsBuildLogging": {
+      "description": "MsBuild logging options.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": "/flp:v=normal"
+    },
+    "MsBuildParameters": {
+      "description": "MsBuild building options.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": "/nologo /maxcpucount /verbosity:minimal"
+    },
+    "Project": {
+      "description": "Project where the commands are going to be applied.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "ExtraArguments": {
+      "description": "Extra parameters will be passed to the selected command.",
+      "valueType": "passThrough",
+      "values": [],
+      "defaultValue": ""
+    },
+    "BatchRestorePackages": {
+      "description": "MsBuild target that restores the packages.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanAllProjects": {
+      "description": "MsBuild target that deletes the binary output directory.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanPackages": {
+      "description": "MsBuild target that deletes the repo-local nuget package directory.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    },
+    "CleanPackagesCache": {
+      "description": "MsBuild target that deletes the user-local nuget package cache.",
+      "valueType": "target",
+      "values": [],
+      "defaultValue": ""
+    }
+  },
+  "commands": {
+    "build": {
+      "description": "Restores packages, builds, runs the tests and builds packages.",
+      "toolName": "msbuild",
+      "lockedSettings": {},
+      "settings": {
+        "MsBuildParameters":"default",
+        "MsBuildLogging":"default"
+      }
+    },
+    "build-s": {
+      "description": "Only builds binaries. It doesn't restore packages.",
+      "toolName": "msbuild",
+      "lockedSettings": {
+        "RestoreDuringBuild": false,
+      },
+      "settings": {
+        "MsBuildLogging":"default",
+        "MsBuildParameters":"default"
+      }
+    },
+    "sync-p": {
+      "description": "Restores all nuget packages for repository",
+      "toolName": "msbuild",
+      "lockedSettings": {
+        "RestoreDuringBuild":  true,
+        "BatchRestorePackages": "default"
+      },
+      "settings": {
+        "MsBuildParameters":"default"
+      }
+    },
+    "clean-b": {
```

Comment:
After talking to @weshaggard  I'm changing the config.json to have more the behavior that you are describing.
I'll put some default options so that you can type `clean`